framework/physics/physics_material/mgxs/mgxs_simple.cc

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#include "framework/physics/physics_material/mgxs/mgxs.h"
+
+namespace opensn
+{
+
+void
+MGXS::Initialize(unsigned int num_groups, double sigma_t)
+{
+  Reset();
+
+  num_groups_ = num_groups;
+  sigma_t_.resize(num_groups, sigma_t);
+  sigma_a_.resize(num_groups, sigma_t);
+  ComputeDiffusionParameters();
+}
+
+void
+MGXS::Initialize(unsigned int num_groups, double sigma_t, double c)
```

Comment:
This entire routine will be replaced with a new version.

Reply:
For my own edification, what is the change going to be?

Reply:
Resolved with MODEL1G.